src/wp-settings.php

```diff
@@ -13,7 +13,9 @@
  *
  * @since 1.0.0
  */
-define( 'WPINC', 'wp-includes' );
+if ( ! defined( 'WPINC' ) ) {
+	define( 'WPINC', 'wp-includes' );
+}
```

Comment:
I think the autoloader needs to be included here too.

```suggestion
}

require_once ABSPATH . 'wp-includes/class-wp-autoload.php';
```

My reasoning is that while the expected entry points to WordPress is either `index.php`, `wp-load.php` or `wp-blog-header.php` it's also possible to use `wp-config.php` as the entrypoint:
```
$ php -r 'include "wp-config.php"; var_dump( $GLOBALS["wp"] );';

Command line code:1:
class WP#1673 (9) {
  public $public_query_vars =>
....
```

While this does still work after this change, this is mostly a side-effect as `wp-settings.php` loads `wp-includes/plugin.php` which directly loads the autoloader, this does seem like a strange unexpected dependency chain, and I'm not sure plugin.php should load it (See other comment)
Without plugin.php loading it, you end up with a fatal (Multisite given, but similar to single site I'm sure)

```
$ php -r 'include "wp-config.php"; var_dump( $GLOBALS["wp"] );';

PHP Fatal error:  Uncaught Error: Class 'WP_Site_Query' not found in wp-includes/ms-site.php:446
```